README.md

```diff
@@ -50,26 +50,85 @@ Responsibilities and roles of the bridge:
 
 ## Usage
 
-### Install Dependencies
+There are two ways to deploy contracts:
+* install and use NodeJS
+* use Docker to deploy 
+
+### Deployment with NodeJS
+
+#### Install Dependencies
 ```bash
 npm install
 ```
-### Deploy
+#### Deploy
 Please the [README.md](deploy/README.md) in the `deploy` folder for instructions and .env file configuration
 
-### Test
+#### Test
 ```bash
 npm test
 ```
 
-### Flatten
+#### Flatten
+Fattened contracts can be used to verify the contract code in a block explorer like BlockScout or Etherscan.
+The following command will prepare flattened version of the contracts:
+
 ```bash
 npm run flatten
 ```
+The flattened contracts can be found in the `flats` directory.
+
+### Deployment in the Docker environment
+[Docker](https://www.docker.com/community-edition) and [Docker Compose](https://docs.docker.com/compose/install/) can be used to deploy contracts without NodeJS installed on the system. 
+If you are on Linux, we recommend you [create a docker group and add your user to it](https://docs.docker.com/install/linux/linux-postinstall/), so that you can use the CLI without `sudo`.
+
+#### Prepare the docker container
+```bash
+docker-compose up --build
+```
+_Note: The container must be rebuild every time when you change the code of contracts or the deployment scripts._
+
+#### Deploy the contracts
+1. Create the `.env` file in the `deploy` directory as it is described in the deployment [README](deploy/README.md).
+2. Run deployment process by
+   ```bash
+   docker-compose run bridge-contracts deploy.sh
+   ```
+   or (in case of Linux system)
+   ```bash
+   ./deploy.sh
+   ```
+
+#### Copy flatten sources (if it is needed)
+1. Discover the container name by
+   ```bash
+   docker-compose images bridge-contracts
+   ```
+2. Use the contaner name in the following comand to copy the flattened code of the contracts to the current working directory. The contracts will be located in the `flats` directory.
```

Comment:
In the following command, use the container name to copy the flattened contracts code to the current working directory. The contracts will be located in the `flats` directory.